lib/AST/ASTContext.cpp

```diff
@@ -3951,8 +3951,17 @@ bool ASTContext::isStandardLibraryTypeBridgedInFoundation(
           nominal == getStringDecl() ||
           nominal == getErrorDecl() ||
           nominal == getAnyHashableDecl() ||
-          // Weird one-off case where CGFloat is bridged to NSNumber.
-          nominal->getName() == Id_CGFloat);
+          // Foundation is a dependency of CoreGraphics, but
```

Comment:
Foundation isn't actually a dependency of CoreGraphics; IIRC the two frameworks are unrelated in iOS, and related the other way on macOS.